.github/workflows/tests.yml

```diff
@@ -37,7 +38,7 @@ jobs:
   test-windows:
     name: Test Code on Windows
     runs-on: windows-latest
-    if: ${{ github.event.label.name == 'needs-windows-test' }}
+    if: contains(github.event.pull_request.labels.*.name, 'needs-windows-test')
```

Comment:
Why is this change necessary?

Reply:
Not sure why, but I could not get the original line to start the test.  I think the original line only sees the labels if they were assigned at the creation of a PR, but not after.